dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -80,46 +75,25 @@ export const LoginRightComponent = () => {
 
 export const AuthForm = () => {
   const navigate = useOutletContext();
+  const { keycloak } = useKeycloak();
 
   const dispatch = useDispatch();
   const navigatePage = (toPage) => {
     dispatch(movePage(toPage, navigate));
   };
+  useEffect(() => {
+    dispatch(authCookies());
+  }, );
```

Comment:
This should fit nicely on one line:
```javascript
  useEffect(() => dispatch(authCookies()));
```

(Ditto line 81, if you feel so inclined.)